### ISSUE TITLE
[Feature Request]: Sort Order for Playlist

### ISSUE THREAD

**Issue by Glint-Eye** [FreeTubeApp/FreeTube#4856]
### Guidelines

- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the feature request I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the feature request I want to file, without success.
- [X] This issue contains only one feature request.

### Problem Description

The current implementation of playlists doesn't offer any sort order options, so each newly added Video goes all the way to the bottom of the playlist. If a list has a lot of videos you always have to scroll all the way down to access your most recently added videos. With a few hundred videos you also have to intermittently load more videos, which makes the process even more cumbersome to access the videos you are probably currently most interested in.

### Proposed Solution

Basically just a reverse sort order Option for playlist.

### Alternatives Considered

None

### Issue Labels

ease of use improvement, improvement to existing feature

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @Glint-Eye im closing this issue because im going to migrate allot of issues over from our past playlist repo. This is one of them but in way more detail.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
I understand why you think that but the playlists feature has been in development for almost 1 year and at some point we have to decide what features need to be in there for the initial release. Otherwise users has to wait longer and longer. 

Improvements will come so dont worry about it :)

**Comment by absidue** [comment]
> Thanks for the info. I guess I just disagree with having postponed this functionality until a later release as it's core to the UX. I will be awaiting the improvements, but for now I have downgraded to 0.19 (using yesterday's old file versions from the cloud, although it might've been possible using 0.20's modified files too).

In the data settings in 0.20.0 you can export the playlists.db file in the old format, the format that 0.19.2 and below use (it moves everything into one big list).

**Comment by absidue** [comment]
> means I have to "Load more videos" until I've loaded ~2000 videos just to see the videos I've recently saved to watch it later

When you are inside a playlist  you can click the magnifying glass icon under the playlist name, to search for videos in that playlist, that should be lot faster than scrolling all the way down.
